<?xml version="1.0"?>
<launch>
  <arg name="robot_name" default="robot1" doc="The name of the robot" />
  <arg name="robot_tag" default="/robot1" doc="The name of the robot" />

  <node pkg="soccer_webots" type="start_single.sh" name="webots_ros_interface" output="screen"
        args="--robot_name $(arg robot_name) --robot_tag $(arg robot_tag)">


  </node>

</launch>